src/components/data-exploration/differential-expression-tool/LaunchPathwayAnalysisModal.jsx

```diff
@@ -0,0 +1,140 @@
+}];
+
+const LaunchPathwayAnalysisModal = (props) => {
+  const { onCancel } = props;
+  const externalServices = { PANTHER: 'pantherdb', ENRICHER: 'enrichr' };
+
+  const [externalService, setExternalService] = useState(externalServices.PANTHER);
+  const [advancedFilteringOpen, setAdvancedFilteringOpen] = useState(false);
+  const [allGenesToggled, setAllGenesToggled] = useState(false);
+
+  return (
+    <>
+      <Modal
+        visible
+        title='Pathway Analysis'
+        width='50%'
+        onCancel={onCancel}
+        // remove next line once the functionality is implemented
+        footer={[<Tooltip key='tooltip' title='Feature coming soon!'><Button disabled>Launch</Button></Tooltip>]}
+        okText='Launch'
+      >
+        <Space direction='vertical' size='large'>
+          {/* display the alert only if there are no filter applied to diff expr */}
+          <Alert
+            type='warning'
+            showIcon
+            message={(
+              <p>
+                You have not performed any filtering on the genes!
+                <Button
+                  type='link'
+                  size='small'
+                  onClick={() => setAdvancedFilteringOpen(!advancedFilteringOpen)}
+                  onKeyPress={() => setAdvancedFilteringOpen(!advancedFilteringOpen)}
+                >
+                  Click here to open the advanced filtering options.
+                </Button>
+              </p>
+            )}
+          />
+
+          <b>External service</b>
+
+          <Radio.Group value={externalService} onChange={(e) => setExternalService(e.target.value)}>
+            {Object.keys(externalServices).map((service) => {
+              const serviceName = externalServices[service];
+              return (<Radio key={service} value={serviceName}>{serviceName}</Radio>);
+            })}
+          </Radio.Group>
+
+          <Space size='large'>
+            <Space direction='vertical'>
+              <b>Species</b>
+
+              <Select style={{ width: 400 }}>
+                {
+                  speciesOptions.map((option) => (
+                    <Select.Option value={option.value}><i>{option.label}</i></Select.Option>
+                  ))
+                }
+              </Select>
+            </Space>
+            <Space
+              style={{ 'margin-left': '5%' }}
+              direction='vertical'
+            >
+              <b>Number of genes</b>
+              <Space>
+                <Radio.Group
+                  value={allGenesToggled}
+                  onChange={(e) => setAllGenesToggled(e.target.value)}
+                >
+                  <Space>
+                    <Radio value>All</Radio>
+                    <Radio value={false}>Top</Radio>
+                  </Space>
+                </Radio.Group>
+                <InputNumber
+                  disabled={allGenesToggled}
+                  size='medium'
+                  style={{ width: '100px' }}
```

Comment:
Does this hardcoded width scale well with smaller screen sizes?

Reply:
I think the modal in general (not only this) doesn't work well with screen sizes, I'll try to adapt everything to smaller screen sizes.

Reply:
Ok, replacing vertical `Space`'s with `Row`'s worked

Reply:
Apparently wrap on a `Space` that is vertical doesn't work